variables.tf

```diff
@@ -11,7 +11,7 @@ variable "repositories" {
 
 variable "chart_version" {
   description = "version of charts"
-  default     = null
+  default     = ">=4.5.10"
```

Comment:
Pin the chart version to `4.5.10`. Not `>=4.5.10`

Reply:
Done